stock_move_force_reservation/i18n/fr.po

```diff
@@ -0,0 +1,155 @@
+"Project-Id-Version: Odoo Server 14.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-07-03 06:30+0000\n"
+"PO-Revision-Date: 2024-07-03 06:30+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,help:stock_move_force_reservation.field_stock_move__picking_state
+msgid ""
+" * Draft: The transfer is not confirmed yet. Reservation doesn't apply.\n"
+" * Waiting another operation: This transfer is waiting for another operation before being ready.\n"
+" * Waiting: The transfer is waiting for the availability of some products.\n"
+"(a) The shipping policy is \"As soon as possible\": no product could be reserved.\n"
+"(b) The shipping policy is \"When all products are ready\": not all the products could be reserved.\n"
+" * Ready: The transfer is ready to be processed.\n"
+"(a) The shipping policy is \"As soon as possible\": at least one product has been reserved.\n"
+"(b) The shipping policy is \"When all products are ready\": all product have been reserved.\n"
+" * Done: The transfer has been processed.\n"
+" * Cancelled: The transfer has been cancelled."
+msgstr ""
+
+#. module: stock_move_force_reservation
+#: model_terms:ir.ui.view,arch_db:stock_move_force_reservation.stock_move_force_reservation_form
+msgid "Cancel"
+msgstr "Annuler"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move_force_reservation__create_uid
+msgid "Created by"
+msgstr "Créé par"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move_force_reservation__create_date
+msgid "Created on"
+msgstr "Créé le"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move_force_reservation__move_id
+msgid "Current Move"
+msgstr "Mouvement actuel"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move_force_reservation__product_id
+msgid "Current Product"
+msgstr "Produit actuel"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move_force_reservation__picking_id
+msgid "Current Transfer"
+msgstr "Transfert actuel"
+
+#. module: stock_move_force_reservation
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move__display_name
+#: model:ir.model.fields,field_description:stock_move_force_reservation.field_stock_move_force_reservation__display_name
+msgid "Display Name"
+msgstr "Nom affiché"
+
+#. module: stock_move_force_reservation
+#: model_terms:ir.ui.view,arch_db:stock_move_force_reservation.stock_picking_form_view
+msgid "Force quantity"
+msgstr "Forcé la quantité"
```

Comment:
```suggestion
msgstr "Forcer la quantité"
```

Reply:
@mathieudelva FYI, translating for existing modules should be done through https://translation.odoo-community.org/